### DISCUSSION TITLE
Why was finetune deleted?

### DISCUSSION THREAD

**Discussion by mboros1** [ggml-org/llama.cpp#10909] [Q&A]
I was investigating the possibility of creating LoRA's with llama.cpp, and saw that it used to exist with the finetune tool but was deleted earlier this year, https://github.com/ggerganov/llama.cpp/commit/be6d7c079173d941b4f784500f9148f46cec2724

Does anyone have insights on why it was removed?

**Comment by hieuhthh** [comment]
After quantize, my generated result is worse. I want to improve this quantized model, like through calibartion, lora, finetune, etc ... Any suggestion or solutions please?
